src/mixpanel-core.js

```diff
@@ -357,7 +357,7 @@ MixpanelLib.prototype._send_request = function(url, data, options, callback) {
     if (!USE_XHR) {
         options.method = 'GET';
     }
-    var use_sendBeacon = sendBeacon && options.transport.toLowerCase() === 'sendbeacon';
+    var use_sendBeacon = sendBeacon && options.method !== 'GET' && options.transport.toLowerCase() === 'sendbeacon';
```

Comment:
Would it be better to instead check `options.method !== 'GET'` since sendBeacon is only for POST? 
Would be useful in case someone had set `api_method` to a non-POST value for whatever reason.

Reply:
Sure, https://github.com/mixpanel/mixpanel-js/pull/247/commits/2590ae27f07bc16d6e1117fe11145573af7ba911. To be fair GET and POST are the only realistic values for this anyway.